pallets/vc-management/src/lib.rs

```diff
@@ -166,6 +166,12 @@ pub mod pallet {
 		#[pallet::weight(195_000_000)]
 		pub fn disable_vc(origin: OriginFor<T>, index: VCIndex) -> DispatchResultWithPostInfo {
 			let who = ensure_signed(origin)?;
+
+			if !VCRegistry::<T>::contains_key(index) {
+				Self::deposit_event(Event::VCNotExist { index });
+				return Ok(().into())
+			}
+
```

Comment:
I don't think we need this, `try_mute` below already does the job.

Btw I think we should emit an error instead of the event in line #185 - F/E will be able to detect this as it's synchronous on-chain logic

Reply:
https://polkadot.js.org/docs/api/cookbook/tx/#how-do-i-get-the-decoded-enum-for-an-extrinsicfailed-event